api/debug.go

```diff
@@ -51,6 +53,7 @@ type DebugAPI struct {
 	client        *requester.CrossSporkClient
 	config        config.Config
 	rateLimiter   RateLimiter
+	chainConfig   *gethParams.ChainConfig
```

Comment:
```suggestion
	evmChainConfig   *gethParams.ChainConfig
```

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/794/commits/3328bfd6bf504423a03b62c10d760c1bc51ccf59